assets/src/edit-story/elements/text/display.js

```diff
@@ -147,7 +132,7 @@ function TextDisplay({
             <BackgroundSpan
               {...props}
               dangerouslySetInnerHTML={{
-                __html: draftMarkupToContent(content, bold),
+                __html: getValidHTML(content),
```

Comment:
Case could be made if we always correctly save `content`, then it will always be valid in the data model and no need to validate it each time. The validation is a bit expensive given the HTML construction/parsing/reformatting/etc.

Reply:
@miina fixed issues elsewhere with malformed HTML, right? Not sure how it happened, but until we're sure it doesn't happen again, I'd rather keep this in (but yes, it's a bit expensive - we might want to memoize it).

Reply:
Yes, I believe all of the issues with malformed HTML have been resolved. And we certainly should have `getValidHTML()` called whenever we call `updateProperties({content})` - there can't be that many of them. So, yeah, we can check that and drop `getValidHTML()` and reading sites.